From c5c3cd92083427ee4e1c5849873fe4db9a506a16 Mon Sep 17 00:00:00 2001
From: Suvidh Subodh Hegde <suvidhsu@qti.qualcomm.com>
Date: Fri, 16 Jan 2026 09:09:43 +0530
Subject: [PATCH] Added feed_board_configuration and feed_netmodel for prplos
 integration in mesh.mcs

Change-Id: I389443aa9830b6792e645ffc0f054aa52e577ca5
Signed-off-by: Suvidh Subodh Hegde <suvidhsu@qti.qualcomm.com>
(cherry picked from commit a6272d9c6ff303cc614b17b61c30aba07ac31745)
---
 feeds.conf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/feeds.conf b/feeds.conf
index 47f65f3aa60d..56058485e6ed 100644
--- a/feeds.conf
+++ b/feeds.conf
@@ -70,6 +70,8 @@ src-link btdaemon ../qca/feeds/btdaemon
 src-link powerboost ../qca/feeds/powerboost
 src-link feed_amx ../qca/feeds/feed_amx
 src-link feed_net_services ../qca/feeds/feed_net_services
+src-link feed_board_configuration ../qca/feeds/feed_board_configuration
+src-link feed_netmodel ../qca/feeds/feed_netmodel
 src-link feed_prplmesh ../qca/feeds/feed_prplmesh
 src-link wapid_cfg80211_internal ../qca/feeds/wapid-cfg80211-internal
 src-link libmmap_telemetry ../qca/feeds/libmmap-telemetry
-- 
2.34.1

